pydomcfg/tests/bathymetry.py

```diff
@@ -147,28 +147,30 @@ def _calc_rmax(depth):
     Parameters
     ----------
     depth: float
-            Bottom depth (units: m).
+        Bottom depth (units: m).
 
     Returns
     -------
     rmax: float
-            Slope steepness value (units: None)
+        Slope steepness value (units: None)
     """
-    depth = depth.reset_index(list(depth.dims))
 
     both_rmax = []
     for dim in depth.dims:
 
-        # (H[0] - H[1]) / (H[0] + H[1])
-        depth_diff = depth.diff(dim)
-        depth_rolling_sum = depth.rolling({dim: 2}).sum().dropna(dim)
-        rmax = depth_diff / depth_rolling_sum
+        rolled = depth.rolling({dim: 2}).construct("tmp_dim")
+
+        # |(H[0] - H[1])| / (H[0] + H[1])
+        # First value is NaN
+        diff = rolled.diff("tmp_dim").squeeze("tmp_dim")
+        rmax = diff / rolled.sum("tmp_dim")
```

Comment:
The alternative discussed in #39 is: `rmax = np.abs(diff) / rolled.sum("tmp_dim")`

Reply:
as per response in #39 - I think @malmans2 solution here is more practical